src/main/java/Main.java

```diff
@@ -0,0 +1,43 @@
+import db.FileWriter;
+import db.FileWriterImpl;
+import db.TransactionReader;
+import db.TransactionReaderImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.CreateReportService;
+import service.CreateReportServiceImpl;
+import service.FruitService;
+import service.FruitServiceImpl;
+import service.OperationStrategy;
+import service.OperationStrategyImpl;
+import service.activities.BalanceHandler;
+import service.activities.OperationHandler;
+import service.activities.PurchaseHandler;
+import service.activities.ReturnHandler;
+import service.activities.SupplyHandler;
+
+public class Main {
+    private static final String filePath =
+            "/Users/nazar5n/IdeaProjects/jv-fruit-shop/src/main/java/storage/file.csv";
```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/b63db0c7-39d4-44de-b860-dc9e88da57ae)

```suggestion
    private static final String filePath =
            "/src/main/resources/file.csv";
```